client/src/components/ClubPage.tsx

```diff
@@ -1,10 +1,45 @@
-import NavBar from "./NavigationBar";
+import React from 'react';
+import NavBar from './NavigationBar';
+import { Link, useParams } from 'react-router-dom';
 
 export default function ClubPage() {
+  // Get the clubName parameter from the URL
+  const { clubName } = useParams();
+
   return (
-    <div>
+    <div style={{ textAlign: 'center' }}>
       <NavBar />
-      Club page
+      {/* Display Club Name */}
+      <h2
+        style={{
+          fontSize: '36px',
+          color: '#333',
+          margin: '20px 0',
+          fontFamily: 'Arial, sans-serif'
+        }}
+      >
+        Club page - {clubName ? decodeURIComponent(clubName) : 'Unknown Club'}
```

Comment:
Weird ChatGPT'd code... Not something we'd need

Reply:
taken out